libs/domain/push-notification/src/services/providers/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './push-provider.service';
+export * from './web-push';
```

Comment:
Push providers should be separated in their own secondary entry points, so they are not part of the main package API but should be imported explicitly, for example `@spryker-oryx/push-notifications/web` for web-push provider.